lib/availability_status_listener.rb

```diff
@@ -51,7 +51,7 @@ def process_event(event)
     options[:last_available_at] = options[:last_checked_at] if options[:availability_status] == 'available'
 
     object.update!(options)
-    raise_event("#{model_class}.update", object.as_json)
+    raise_event("#{model_class}.update", object.as_json, event.headers)
```

Comment:
@lindgrenj6 does `event.headers` return nil or raise exception if not set in operations worker?

Reply:
No, it is just an empty hash luckily.